adapters/repos/db/replication.go

```diff
@@ -214,16 +214,17 @@ func (i *Index) ReplicateReferences(ctx context.Context, shard, requestID string
 }
 
 func (i *Index) CommitReplication(shard, requestID string) interface{} {
-	localShard := retryGetLocalShard(i, shard)
-	if localShard == nil {
+	localShard, err := i.getOrInitLocalShard(context.Background(), shard)
+	// TODO-RAFT no error response on error?
+	if err != nil {
 		return nil
 	}
 	return localShard.commitReplication(context.Background(), requestID, &i.backupMutex)
 }
 
 func (i *Index) AbortReplication(shard, requestID string) interface{} {
-	localShard := retryGetLocalShard(i, shard)
-	if localShard == nil {
+	localShard, err := i.getOrInitLocalShard(context.Background(), shard)
```

Comment:
If we abort the operation and the shard does not exist, then  we don't need to create the shard, and it is safe to simply return success.